ci/vm/provision1.sh

```diff
@@ -32,7 +32,11 @@ install_mainline_kernel() {
     add-apt-repository ppa:cappelikan/ppa    
     apt-get update &&
       DEBIAN_FRONTEND=noninteractive apt-get install -yq mainline
-    mainline install-latest
+    # Install 2 minor versions back from the latest available (X.Y-2.Z)
```

Comment:
maybe simpler to pick a specific one and manually update that value when needed? for instance, if two more version are added then we would end up picking the currently non-working one?

Reply:
Agree, let's do https://github.com/openucx/xpmem/pull/41 instead.